### PR TITLE
Add handgrading to ultimate submission score download csv

### LINKED ISSUE #344: Include handgrading score when downloading ultimate submission csv
Only applicable if the project has a HandgradingRubric. 

There should be two new columns, "Handgrding Total Points" and "Handgrading Total Points Possible". These columns should be filled by groups that have a HandgradingResult object, and only if it is marked as finished_grading. Otherwise, the groups handgrading column entries should be empty.

### PR DESCRIPTION
**Pull request by ffuxa**
Fixes #344

### CODE REVIEW

File: autograder/rest_api/serialize_ultimate_submission_results.py [hunk 1]
```diff
@@ -8,7 +8,8 @@
 
 
 def serialize_ultimate_submission_results(ultimate_submissions: Iterable[SubmissionResultFeedback],
```
[the remark this answers is not in the document]
Comment (ffuxa) [reply autograder/rest_api/serialize_ultimate_submission_results.py]: Added tests ✅

File: autograder/rest_api/serialize_ultimate_submission_results.py [hunk 2]
```diff
@@ -86,6 +94,21 @@ def _get_submission_data_with_results(submission_fdbk: SubmissionResultFeedback,
...
+                submission_fdbk.submission.handgrading_result.finished_grading)
+        )
+
+        if handgrading_result_available:
+            handgrading_result = submission_fdbk.submission.handgrading_result
+            submission_results['handgrading_total_points'] = handgrading_result.total_points
+            submission_results['handgrading_total_points_possible'] = (
+                handgrading_result.total_points_possible)
+        else:
+            submission_results['handgrading_total_points'] = ''
```
Comment (james-perretta) [inline autograder/rest_api/serialize_ultimate_submission_results.py]: Why empty string instead of None?

Comment (ffuxa) [reply autograder/rest_api/serialize_ultimate_submission_results.py]: The goal was to have the CSV file have an empty entry on the Handgrading Total Points column for this case. If setting it equal to None does the same effect then I can do that instead.

Comment (james-perretta) [reply autograder/rest_api/serialize_ultimate_submission_results.py]: Makes sense. If None accomplishes the same thing, I think None has a slightly clearer meaning in the code and should be preferred.

**Pull request merged**